tests/commands/run_test.py

```diff
@@ -292,6 +292,53 @@ def test_always_run_alt_config(
     )
 
 
+def test_hook_verbose_enabled(
+        cap_out, repo_with_passing_hook, mock_out_store_directory
+):
+    with modify_config() as config:
+        config['repos'][0]['hooks'][0]['always_run'] = True
+        config['repos'][0]['hooks'][0]['verbose'] = True
+
+    _test_run(
+        cap_out,
+        repo_with_passing_hook,
+        {},
+        (b'Hello World',),
+        0,
+        stage=False,
+    )
```

Comment:
this test is probably sufficient, the others I don't think are super necessary (we already test all three of those scenarios)